setup.py

```diff
@@ -62,6 +62,6 @@
                 "sphinx_rtd_theme",
             ],
             "dali": ["dali-dataset==1.1"],
-            "haydn_op20": ["music21"],
+            "haydn_op20": ["music21==6.7.1"],
```

Comment:
Generally bad idea in a library to pin exactly. Maybe use <=6.7.1?

Reply:
the latest version is what's breaking it :(